docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -0,0 +1,149 @@
+[[get-snapshot-api]]
+=== Get snapshot API
+++++
+<titleabbrev>Get snapshot</titleabbrev>
+++++
+
+Returns information about one or more snapshots from one or more registered repositories.
```

Comment:
Per Deb, we try to use "retrieve" in place of "return" as it's a bit more user-centric. I also don't think the repos clause is needed here, but feel free to ignore if wanted.
```suggestion
Retrieves information about one or more snapshots.
```